sandbox/OpenIddict.Sandbox.Avalonia.Client/OpenIddict.Sandbox.Avalonia.Client/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using OpenIddict.Sandbox.Avalonia.Client.ViewModels;
+using OpenIddict.Sandbox.Avalonia.Client.Views;
+
+namespace OpenIddict.Sandbox.Avalonia.Client;
+
+public static class ServiceCollectionExtensions
+{
+
+    public static IServiceCollection AddApp(this IServiceCollection services)
```

Comment:
Looks like that extension isn't very useful. Please consider eliminating all those tiny files to make the sample lighter.